CRVResponse/fcl/prolog_v11.fcl

```diff
@@ -51,19 +51,30 @@ BEGIN_PROLOG
                                    "CRVConditions/v6_0/LookupTable_2100_1",  //C3
                                    "CRVConditions/v6_0/LookupTable_1550_1"]  //C4
 
-                                //number of photons per MeV visible energy deposited
-                                //for 68 PE/SiPM @ 1 m away from SiPM (Test beam June 2017)
-                                //using the pulse height calibration, this value gives 45 PE/SiPM @ 1 m away from SiPM
-      scintillationYields       : [39400,39400,39400,39400,39400,39400,  //R1...6
-                                   39400,39400,39400,                    //L1...3
-                                   39400,39400,39400,39400,39400,        //T1...5
-                                   39400,39400,                          //E1,E2
-                                   39400,                                //U
-                                   39400,39400,39400,39400,              //D1...4
-                                   0,0,39400,39400]                      //C1...4
+                                //all measurements were done at 1m away from SiPM with a reference date of 12/2023
+                                //CRV-Top (1.8mm) modules: 39794 - based on comparison between Wideband measurements and fined-tuned simulation
+                                //                         (all counters come from scintillator batch 1)
+                                //CRV-Top (1.4mm) modules: 30442 - based on comparions between UVA measurements of 1.4mm and 1.8mm CRV-Top modules (adjusted for age)
+                                //                         (most counters come from scintillator batch 1)
+                                //                         assuming 3% aging per year
+                                //                         T (1.4mm) measured 2/2022 at 44.5 PEs, 1.83 years to 12/2023, aging factor 0.946, --> 42.1 PEs
+                                //                         T (1.8mm) measured 8/2022 at 57.3 PEs, 1.33 years to 12/2023, aging factor 0.960, --> 55.0 PEs
+                                //                         --> photon yield ratio: 0.765
+                                //CRV-TS modules:          30442 - same as CRV-Top (1.4mm), because all counters come from scintillator batch 1
```

Comment:
I think these estimates are fine for now, but we'll have to re-measure the light yields for every channel and put them in the database. Is that the plan?

Reply:
I suggest that defaults be for Run 1 and, if needed, you also provide standard configurations for the extracted position.  That last part does not need to be in this PR

Reply:
Once we have all measurements, we can put them in the database. For now, the average value for each sector is sufficient (for both the full geometry and the extracted position).